app/src/main/java/com/cmput301f20t21/bookfriends/ui/add/AddEditActivity.java

```diff
@@ -188,11 +190,11 @@ private void loadBookInformation() {
         titleLayout.getEditText().setText(editBook.getTitle());
         authorLayout.getEditText().setText(editBook.getAuthor());
         descriptionLayout.getEditText().setText(editBook.getDescription());
-        Uri imageUri = editBook.getImageUri();
-        if (imageUri != null) {
-            Glide.with(this).load(imageUri).into(bookImage);
-            bookImageUri = imageUri;
-        }
+        StorageReference storageReference = FirebaseStorage.getInstance().getReference(editBook.getId() + "cover");
```

Comment:
we could abstract `editBook.getId() + "cover"` to `editBook.getCoverImageName()`